spec/integration/.kitchen.yml

```diff
@@ -0,0 +1,19 @@
+---
+driver:
+  name: docker
+  image: centos:6.7
+  use_sudo: true
```

Comment:
The `use_sudo` argument will fail to build on local OSX hosts because docker shell variables are set at the user level. Dunno a good solution around this without some hackery